### PR TITLE
Updating Genesis Tools

### PR DESCRIPTION
**Pull request by ravenac95**

### CODE REVIEW

File: .github/staking_config.yaml [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# Staking Ledger Allocations in YAML so we can comment as needed
+
+# This file is a simple "key": "value" where key is the github name and the
```
Comment (ravenac95) [inline .github/staking_config.yaml:3]: need to fix the comment here. This has changed significantly

File: go/go.mod [hunk 2]
```diff
@@ -11,9 +11,8 @@ replace (
 )
 
 require (
-	github.com/dgraph-io/badger/v2 v2.0.3 // indirect
 	// https://github.com/oasisprotocol/oasis-core/releases/tag/v20.9
```
Comment (kostko) [inline go/go.mod:14]: Drop this comment :-)

Comment (ravenac95) [reply go/go.mod:14]: Oh haha! thanks :)

**Comment by ravenac95** [comment]
@kostko would appreciate a review, I realize there's a lot of jank here but I do want to make sure our calculations for everything are correct. Nearing the end I ended up with more frankenstein-ing as I added the CSV support in the last minute. So the state of the code isn't so great. However, the parts that do the Accounting for Genesis should be correct.

**Comment by kostko** [comment]
> however, the parts that do the Accounting for Genesis should be correct.

At first glance it seems ok.

File: go/genesis-tools/stakinggenesis/accounting.go [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+	if err != nil {
+		return err
+	}
+	err = a.ledger[to].Escrow.Active.TotalShares.Add(preciseAmount)
+	if err != nil {
+		return err
+	}
+
+	// Ensure the commission schedule is set since this account is getting
+	// delegations. This is currently set to a bound of 0-20% and a starting rate of 5%
```
Comment (ravenac95) [inline go/genesis-tools/stakinggenesis/accounting.go:122]:
```suggestion
	// delegations.
```

**Review by Afauvre — APPROVED** [review]

**Pull request merged**